pkgs/by-name/nr/nrf-udev/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gitUpdater
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "nrf-udev";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "NordicSemiconductor";
+    repo = "nrf-udev";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-bEIAsz9ZwX6RTzhv5/waFZ5a3KlnwX4kQs29+475zN0=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    cp -r nrf-udev_1.0.1-all/lib $out
```

Comment:
```suggestion
    cp -r nrf-udev_${version}-all/lib $out
```
or 
```suggestion
    cp -r nrf-udev_*/lib $out
```
?

Reply:
Last one looks like the most stable, thanks!